src/sonic-yang-models/yang-models/sonic-buffer-profile.yang

```diff
@@ -0,0 +1,67 @@
+module sonic-buffer-profile {
+	namespace "http://github.com/Azure/sonic-buffer-profile";
+	prefix bpf;
+
+	import sonic-buffer-pool {
+		prefix bpl;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC BUFFER PROFILE";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+
+	container sonic-buffer-profile {
+
+		container BUFFER_PROFILE {
+
+			list BUFFER_PROFILE_LIST {
+				key "name";
+
+				leaf name {
+					type string;
+				}
+
+				leaf static_th {
+					type uint64;
+				}
+
+				leaf dynamic_th {
+					type int64;
+				}
+
+				leaf size {
+					type uint64;
```

Comment:
Please refer the below SAI spec and update the default values for fields accordingly.

https://github.com/opencomputeproject/SAI/blob/master/inc/saibuffer.h